### DISCUSSION TITLE
Faster large file transfers between peers

### DISCUSSION THREAD

**Discussion by agingo** [paullouisageneau/libdatachannel#1296] [General]
When transferring / chunking large file transfers between peers we are experiencing low transfer speeds. Small messages are fine but larger messages take a long time to transfer. Other than buffer sizes and doing parallel transfers how do you suggest speeding up large file transfers?

We prefer to use RTC to transfer large files as we've already optimized it to punch through strict firewalls.

Thanks,

**Comment by paullouisageneau** [comment]
Small messages are more efficient in general, but it's hard to give you an answer without specific numbers. What do you mean by larger messages? Why do you need to send larger messages if you chunk the files? Simply choose a message size that gives good performance (65K is typically good for both compatibility and performance).
